chart/values.yaml

```diff
@@ -639,7 +639,7 @@ jupyterhub:
       component: singleuser-server
     image:
       name: registry.gitlab.com/aiacademy/docker-stacks/aia-notebook
-      tag: latest-cpu
+      tag: no-existing-tag
```

Comment:
The image is no longer used. We can change to the latest base-notebook.

```yaml
name: infuseai/docker-stacks
tag: base-notebook-1eb85fe1
```

Reply:
This is not really used in the template. It is always replaced in the `jupyterhub_profiles.py`